src/Sylius/Bundle/ChannelBundle/Form/Type/ChannelEntityChoiceType.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ChannelBundle\Form\Type;
+
+/**
+ * Channel choice form type.
+ *
+ * @author Paweł Jędrzejewski <pjedrzejewski@diweb.pl>
+ */
+class ChannelEntityChoiceType extends ChannelChoiceType
```

Comment:
This can be removed in favour of `Sylius/Bundle/ResourceBundle/Form/Type/ResourceChoiceType`